page.css

```diff
@@ -0,0 +1,98 @@
+
+html,
+body {
+    background-color: #FFE1E9;
+    font-family: sans-serif;
+}
+
+h1 {
+    font-size: 60px;
+}
+
+@media only screen and (min-width: 375px) {
+
+    .body__image {
+        display: flex;
+        height: 150px;
+
+    }
+    
+    .page_container {
+        display: block;
+        background-color: white;
+    }
+    
+    
+    .body_maincard {
+        background-color: white;
+    }
+    
+    .body__top_layout {
+         background-color: white;
+         padding: 50px;
+    
+    }
+    
+    p {
+        font-size: 33px; 
+    }
+
+
+    h1 {
+    display: block;
+    }
+
+    textarea {
+        border-radius: 15px;
+        width: 99%;
+        font-size: 35px;
+        text-align: center;
+        line-height: 80px;
+        height:100px;
+    }
+
+    .body__signedin {
+        padding: 30px;
+        font-size: 35px;
+        display: flex;
+    }
+    
+    input[type=checkbox] {
+        transform: scale(3.8);
```

Comment:
do we really need scale here?

Reply:
Yes. There are several best practice solutions to change checkbox size as far as I know

We can use width and height too